model/account/type.go

```diff
@@ -66,6 +66,7 @@ type AccountType struct {
 	Slug                  string            `json:"slug,omitempty"`
 	Secret                interface{}       `json:"secret,omitempty"`
 	SkipRedirectURI       bool              `json:"skip_redirect_uri_on_authorize,omitempty"`
+	SkipState             bool              `json:"skip_state_on_token,omitempty"`
```

Comment:
Can we upgrade the associated doc to list those new params?

Reply:
Done: https://github.com/cozy/cozy-stack/commit/11cb069b277c3340909c4076270d756f422714ab